CHANGELOG.md

```diff
@@ -125,6 +125,10 @@
 * Make `trailing_closure` correctable.  
   [KS1019](https://github.com/KS1019/)
 
+* Add new `final_class` rule to prefer `static` over `final class` methods.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown:
```suggestion
* Add new `final_class` rule to prefer `static` over `final class` declarations.  
```